From b52e76432fe23490b891884f3a66504101c2a5bd Mon Sep 17 00:00:00 2001
Message-Id: <b52e76432fe23490b891884f3a66504101c2a5bd.1402480818.git.baruch@tkos.co.il>
From: Baruch Siach <baruch@tkos.co.il>
Date: Wed, 11 Jun 2014 12:56:15 +0300
Subject: [PATCH] configure.in: fix external libblkid test for static link

External libblkid needs -luuid when linking statically. Use pkg-config to
determine it correctly. Adding --static doesn't hurt even for dynamic
linking.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---

Patch status: sent upstream: http://patchwork.ozlabs.org/patch/358584/

 configure.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.in b/configure.in
index 448a2926776a..ec6068320878 100644
--- a/configure.in
+++ b/configure.in
@@ -551,7 +551,7 @@ then
 		[LIBBLKID=`$PKG_CONFIG --libs blkid`;
 		 STATIC_LIBBLKID=`$PKG_CONFIG --static --libs blkid`],
 		[AC_MSG_ERROR([external blkid library not found])],
-		[$LIBBLKID])
+		[`$PKG_CONFIG --static --libs blkid`])
 	BLKID_CMT=#
 	AC_MSG_RESULT([Disabling private blkid library])
 else
-- 
2.0.0

